modules/virt-adding-vtpm-to-vm.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-vtpm-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-adding-vtpm-to-vm_{context}"]
+= Adding a vTPM device to a virtual machine
+
+Adding a virtual Technical Platform Module (vTPM) device to a virtual machine
+(VM) allows you to run a VM created from a Windows 11 image without a physical
+TPM device. A vTPM device also temporarily stores secrets for that VM.
+
+.Procedure
+
+. Open the VM manifest in your default editor by running the following command:
```

Comment:
If we're using oc edit here, it's better to rephrase to:
Run the following command to update the virtual machine configuration: